### PR TITLE
feature  - add support for `getSession` and `setSession`

### PR DESCRIPTION
**Pull request by andrejpavlovic**
It seems like ios implicitly sets session when making calls via `send` method while android doesn't. In order to make it more consistent I've exposed cross-platform `getSession` and `setSession` methods, so that javascript client can set session data as needed.

e.g.
```typescript
const result = await Gigya.send('accounts.notifySocialLogin', params)
await Gigya.setSession(
  result.sessionInfo.sessionToken,
  result.sessionInfo.sessionSecret
)
await this.getAccountInfo()
```

### CODE REVIEW

File: ios/GigyaSdkWrapper.swift
Comment (tal-mi) [inline ios/GigyaSdkWrapper.swift:35]: A public model **GigyaSession** is already present within the Swift SDK. No need to create a new one.

File: ios/GigyaSdkWrapper.swift
Comment (tal-mi) [inline ios/GigyaSdkWrapper.swift:74]: Expiration should be double and no NSNumber.

File: ios/GigyaSdkWrapper.swift
Comment (tal-mi) [inline ios/GigyaSdkWrapper.swift:123]: Instead of adding a new function to create a dictionary from the session object just do:

`let sessionAsMap = [“token”: session?.token, “secret”: session?.secret, “sessionExpirationTimestamp”: session.session?.sessionExpirationTimestamp]
`

No need for a conversion function.

File: ios/GigyaSdkWrapper.swift
Comment (tal-mi) [inline ios/GigyaSdkWrapper.swift:340]: Conversion function is not needed - remove.

**Review by tal-mi — COMMENTED** [review]
Good additions. 
Minor changes are required before approval.

File: ios/GigyaSdkWrapper.swift
Comment (andrejpavlovic) [reply ios/GigyaSdkWrapper.swift]: Removed.

File: ios/GigyaSdkWrapper.swift
Comment (andrejpavlovic) [reply ios/GigyaSdkWrapper.swift]: Updated.

File: ios/GigyaSdkWrapper.swift
Comment (andrejpavlovic) [reply ios/GigyaSdkWrapper.swift:117]: I still need the conversion function since I'm using it in two different places.

File: ios/GigyaSdkWrapper.swift
Comment (andrejpavlovic) [reply ios/GigyaSdkWrapper.swift:334]: Function is being called from getSession and setSession, so it would make sense to me to leave it here.

Comment (andrejpavlovic) [reply ios/GigyaSdkWrapper.swift:334]: I've modified it to use create the dictionary directly instead of relying on jsonEncoder

**Review by tal-mi — APPROVED** [review]
Approved

**Pull request merged**
